ResearchJournal.tex

```diff
@@ -0,0 +1,58 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Can Video games be tailored to their User by procedurally creating challenging emergent gameplay and narrative experiences.}
+
+% Please do not change the subtitle
+\subtitle{COMP130 - Research Journal}
+
+% Please put your student number in the author field
+\author{1507729}
+
+\begin{document}
+
+\maketitle
+
+\abstract{}
+
+\section{Can Users Trust Computer Agents?}
+
+It's very common in video games to see a computer agent or non player character (NPC). NPC characters are in computer terms agents which will perform certain actions depending on predetermined rule sets, a simple example might be if an NPC can see the player he will attack them. However if you wished to build a procedural narrative experience it's important that NPC characters who are relevant to your plot lines are capable of more complex or varied actions. An upcoming example is the Middle-earth: Shadow of war which features procedurally created NPC characters with unique behaviours that allow users to create their own narrative experiences. \cite{monolith2017shadow}. It's worth noting that various studies have looked into creating a sense of trust with NPC characters and looking at ways in which users interact with NPC's. \cite{do2016trust} On the opposite end of this spectrum it's also worth noting ways in which NPC's or game agents could decieve or lie to a player, by limiting the knowledge of what the NPC is aware of it's possible to get a different response. Once such example of this research is shown here \cite{cowling2015emergent}.
```

Comment:
Interesting to consider whether techniques for bluffing in card games really can be applied to complex NPC behaviours in video games, or indeed whether such techniques are really needed in that domain. Worth exploring more.